sync/mocks.go

```diff
@@ -0,0 +1,52 @@
+package sync
```

Comment:
I'd put that in a `test` file as they are not compiled into the final bin

Reply:
i forgot, its not in the test file because im using it in the cmd package